src/Microsoft.SqlTools.ServiceLayer/DisasterRecovery/CommonUtilities.cs

```diff
@@ -376,10 +375,7 @@ public static string[] GetAssociatedFilePaths(ServerConnection connection, strin
 
             foreach (DataRow currentRow in dataTable.Rows)
             {
-                var primaryFolder = Path.GetDirectoryName(primaryFilePath);
```

Comment:
This previous code was a bit of SSMS logic that didn't quite translate without all the extra SSMS classes and checks.